src/test/java/fi/fmi/avi/util/BulletinHeadingDecoderTest.java

```diff
@@ -62,48 +61,69 @@ public void decode_bulletin_headings(final String input, final BulletinHeading e
+                        .setBulletinAugmentationNumber(1)
+                        .setBulletinAugmentationIndicator("CCA")
+                        .build(),
+                        EXTENDED_AUGMENTATION_IDENTIFIERS}, //
+                new Object[]{"FTFI31 EFLK 250200 RTD", TAF_BULLETIN_HEADING.toBuilder()
+                        .setType(BulletinHeading.Type.DELAYED)
+                        .setBulletinAugmentationNumber(1)
+                        .setBulletinAugmentationIndicator("RRA")
+                        .build(),
+                        EXTENDED_AUGMENTATION_IDENTIFIERS}, //
+                new Object[]{"FTFI31 EFLK 250200 AMD", TAF_BULLETIN_HEADING.toBuilder()
+                        .setType(BulletinHeading.Type.AMENDED)
+                        .setBulletinAugmentationNumber(1)
+                        .setBulletinAugmentationIndicator("AAA")
+                        .build(),
+                        EXTENDED_AUGMENTATION_IDENTIFIERS}, //
+                new Object[]{"FTFI31 EFLK 250200 INVALID", TAF_BULLETIN_HEADING.toBuilder()
+                        .setType(BulletinHeading.Type.NORMAL)
+                        .setBulletinAugmentationIndicator("")
+                        .build(),
+                        EXTENDED_AUGMENTATION_IDENTIFIERS}, //
                 // SIGMETs
-                new Object[] { "WSFI31 EFLK 231008", SIGMET_BULLETIN_HEADING, ConversionHints.EMPTY },
-                new Object[] { "WSFI32 AAAA 231008", SIGMET_BULLETIN_HEADING.toBuilder().setBulletinNumber(32).setLocationIndicator("AAAA").build(),
-                        ConversionHints.EMPTY } };
+                new Object[]{"WSFI31 EFLK 231008", SIGMET_BULLETIN_HEADING, ConversionHints.EMPTY},
+                new Object[]{"WSFI32 AAAA 231008", SIGMET_BULLETIN_HEADING.toBuilder().setBulletinNumber(32).setLocationIndicator("AAAA").build(),
+                        ConversionHints.EMPTY}};
     }
 
     @Test
     @Parameters
     public void decode_bulletin_headings_throws_exception_on_error(final String input, final ConversionHints conversionHints,
-            final Class<Throwable> expectedException) {
+                                                                   final Class<Throwable> expectedException) {
         assertThatExceptionOfType(expectedException).isThrownBy(() -> BulletinHeadingDecoder.decode(input, conversionHints));
     }
 
     public Object parametersForDecode_bulletin_headings_throws_exception_on_error() {
-        return new Object[] {//
+        return new Object[]{//
                 // Extended augmentation indicators
-                new Object[] { "FTFI31 EFLK 250200 INVALID", EXTENDED_AUGMENTATION_IDENTIFIERS, IllegalArgumentException.class }, //
-                new Object[] { "FTFI31 EFLK 250200 RTD", ConversionHints.EMPTY, IllegalArgumentException.class }, //
+                new Object[]{"FTFI31 EFLK 250200 INVALID", ConversionHints.EMPTY, IllegalArgumentException.class}, //
```

Comment:
I would guess that the original intention in this test line was to ensure, exception is thrown whenever `BulletinHeadingIndicatorInterpreter` returns an invalid value?

Reply:
True, I added this test case back.